src/main/antlr4/imports/mysql_alter_table.g4

```diff
@@ -65,3 +65,9 @@ ignored_alter_specifications:
     ;
   algorithm_type: DEFAULT | INPLACE | COPY;
   lock_type: DEFAULT | NONE | SHARED | EXCLUSIVE;
+
+alter_ordering: alter_ordering_column (ASC|DESC)?;
+alter_ordering_column:
+    name '.' name '.' name
```

Comment:
not quite, there's three forms of column names that are *technically* acceptable:

```
`col`
`tbl`.`col`
`db`.`col`
```

In this case, it doesn't matter if I give them aliased names, as I'm parsing and throwing away all of this stuff.